cmdstanpy/model.py

```diff
@@ -269,32 +279,23 @@ def src_info(self) -> Dict[str, Any]:
         If stanc is older than 2.27 or if the stan
         file cannot be found, returns an empty dictionary.
         """
-        result: Dict[str, Any] = {}
-        if self.stan_file is None:
-            return result
-        try:
-            cmd = (
-                [os.path.join(cmdstan_path(), 'bin', 'stanc' + EXTENSION)]
-                # handle include-paths, allow-undefined etc
-                + self._compiler_options.compose_stanc()
-                + [
-                    '--info',
-                    str(self.stan_file),
-                ]
-            )
-            proc = subprocess.run(
-                cmd, capture_output=True, text=True, check=True
+        if self.stan_file is None or cmdstan_version_before(2, 27):
+            return {}
+        cmd = (
+            [os.path.join(cmdstan_path(), 'bin', 'stanc' + EXTENSION)]
+            # handle include-paths, allow-undefined etc
+            + self._compiler_options.compose_stanc()
+            + ['--info', str(self.stan_file)]
+        )
+        # pylint: disable=subprocess-run-check
+        proc = subprocess.run(cmd, capture_output=True, text=True)
```

Comment:
I think you can make this pylint go away even by specifying `check=False`, which I prefer to the comment.